cmd/bucket-lifecycle.go

```diff
@@ -70,6 +71,28 @@ func NewLifecycleSys() *LifecycleSys {
 	return &LifecycleSys{}
 }
 
+func ilmTrace(startTime time.Time, duration time.Duration, oi ObjectInfo, event string) madmin.TraceInfo {
+	return madmin.TraceInfo{
+		TraceType: madmin.TraceILM,
+		Time:      startTime,
+		NodeName:  globalLocalNodeName,
+		FuncName:  event,
```

Comment:
An ILM action may be called from multiple code paths. For example, ILMExpiry may be called during scanning or when serving a HEAD/GET object request on an object version past its expiry. We can use `madmin.TraceInfo.Message` to pass the actual function name that invoked this ILM action.

```suggestion
		FuncName:  event,
		Message, getSource(3),	
```
Note: 3 in `getSource(3)` above depends on how many functions deep `ilmTrace` is from the function calling `globalLifecycleSys.Trace`. Please correct this based on your local testing.